src/language-handlebars/printer-glimmer.js

```diff
@@ -289,8 +291,21 @@ function print(path, options, print) {
       ]);
     }
     case "MustacheCommentStatement": {
+      const rawNodeContent = textLines[n.loc.start.line - 1].slice(
+        n.loc.start.column,
+        n.loc.end.column
+      );
```

Comment:
start/end don't have to on same line, right?

**Prettier pr-9082**
[Playground link](https://deploy-preview-9082--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuExgD8CEACAsgVwBsYBLAHSm20JKjnQF8GQAaECAB1OgGdlQAhgCchEAO4AFYQj4oBhMQICefNgCMhAsAGs4MAMocttAObIYQ-HDZwAtmrgATR04AyAqCfwCTcAGIQQrYCMKSeyCAC+DAQrCAAFjC2hADq8STwPEZgcPoyGSQAbhlKEWA8qiC0PHBCMBKaJsHIAGbyNWwAVjwAHgBCmjp6+gK2cK60cK3t1iDdPfqmhHAAivgQ8NOEHSBGQjVCESY0tmOHbBxCtDApJI4w8cgADBeiNSmaHBGXcAeFU2wAI7reANTiySI8AC0dCcTjiQjgwJIiIaPmaSDa21mNVsJHMlhxS1WIKmmJmbBgAjUt3ujyQACZKZoSDRPABhCCnAQRX4AVji+BqABVqbIsTtClYAJJQFywfRgK5cACCcv0MCUyy2NSYQA)
```sh
--parser glimmer
--print-width 0
```

**Input:**
```hbs
{{~! Multi
  line~}}
```

**Output:**
```hbs
{{~! Multi
  line}}
```